### PR TITLE
Corrections, edits to Record, Play API's

### PR DESCRIPTION
**Pull request by tituscha**
Some edits, corrections which came up during our team discussion.

### CODE REVIEW

File: api-reference/beta/api/call-playprompt.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ Play a prompt in the call.
 
 For more information about how to handle operations, see [commsOperation](../resources/commsoperation.md)
 
+#### Note
```
Comment (Lauragra) [inline api-reference/beta/api/call-playprompt.md]: This is not how we format notes. You can use this syntax for a note that you want to be formatted as a colored box on the site:
> [!NOTE]
> Note text.

File: api-reference/beta/api/call-playprompt.md [hunk 2]
```diff
@@ -41,9 +43,9 @@ In the request body, provide a JSON object with the following parameters.
 
 | Parameter      | Type    |Description|
 |:---------------|:--------|:----------|
-|prompts|[prompt](../resources/prompt.md) collection| Currently only a single prompt and of type [MediaPrompt](../resources/mediaprompt.md) is supported.|
+|prompts|MediaPrompt collection| Currently only a single prompt and of type [MediaPrompt](../resources/mediaprompt.md) is supported.|
 |loop|bool| The loop value. true indicates to loop infinitely. The default value is false. |
-|clientContext|String|The client context.|
+|clientContext|String|Unique Client Context string. Max limit is 256 chars.|
```
Comment (Lauragra) [inline api-reference/beta/api/call-playprompt.md]: I don't think this needs to be capped. Also "Max limit" is duplicative. Revise as: "Unique client context string. Can have a maximum of 256 characters.

File: api-reference/beta/api/call-record.md [hunk 3]
```diff
@@ -44,15 +44,15 @@ In the request body, provide a JSON object with the following parameters.
...
+|initialSilenceTimeoutInSeconds | Int32| Maximum initial silence (user silence) allowed from the time we start the record operation before we timeout and fail the operation. If we are playing a prompt, then this timer starts after prompt finishes. Default = 5 seconds, Min = 1 second, Max = 300 seconds |
 |maxSilenceTimeoutInSeconds|Int32| Maximum silence (pause) time allowed after a user has started speaking. Default = 5 seconds, Min = 1 second, Max = 300 seconds.|
 |maxRecordDurationInSeconds|Int32| Max duration for a record operation before stopping recording. Default = 5 seconds, Min = 1 second, Max = 300 seconds.|
 |playBeep|Boolean| If true, plays a beep to indicate to the user that they can start recording their message. Default = true.|
 |stopTones|String collection|Stop tones specified to end recording.|
-|clientContext|String|The client context.|
+|clientContext|String|Unique Client Context string. Max limit is 256 chars.|
```
Comment (Lauragra) [inline api-reference/beta/api/call-record.md]: Revise here as per earlier comment.

File: api-reference/beta/api/call-playprompt.md (hunk 2 above)
Comment (tituscha) [reply api-reference/beta/api/call-playprompt.md]: Changed it.

File: api-reference/beta/api/call-record.md (hunk 3 above)
Comment (tituscha) [reply api-reference/beta/api/call-record.md]: Change it.

**Comment by tituscha** [comment]
@Lauragra - Hi Larura, this is ready to be merged. Is there something that would be needed from my end.

**Pull request merged**